### ISSUE TITLE
Display Issue with ESP32-WROOM-32U

### ISSUE THREAD

**Issue by marosell** [alf45tar/PedalinoMini#17]
I built and uploaded the esp32dev build to my ESP32-WROOM-32U (ESP32-DevKitC-32U) and it successfully uploaded. However, the screen shows static that slowly moves up a few lines at a time. I can confirm that the board briefly broadcasted a wifi name similar to "Pedalino", however, I pretty quickly used ESPTouch to provide the board with my wifi password.

![IMG_8665](https://user-images.githubusercontent.com/1125586/64073165-04c20500-cc68-11e9-81a0-d33905ef9531.jpg)

I can now confirm by looking at my router that the board is connected to my wifi (device ID shows "ESP_09B3CC" is connected, but I am unable to navigate to ESP_09B3CC.local in my web browser, or the ip address to pull up the web user interface.

Any idea how I can fix the screen, or how I can find the web address without the screen?

**Comment by underwoodblog** [comment]
Do you power it via the microusbcable? Try to change it with a quality one or power it from a PSU.

**Comment by marosell** [comment]
> Do you power it via the microusbcable? Try to change it with a quality one or power it from a PSU.

I do power via USB micro cable, from computer. I tried different cables and different usb ports and it did not solve the issue. The issue was that I purchased a SH1106 OLED display and not a SSD1306 display, the latter is what is set as default in the build unless you use the TTGO_T_EIGHT board (which is unavailable).

Changing some of the if/else statements in the DisplayOLED.H file will solve the issue if using an SH1106 OLED display, however, it seems like there could be a way to either detect the display or make it easier to select the display before uploading to the board?
